### ISSUE TITLE
Clan\Group Endpoint Webhooks

### ISSUE THREAD

**Issue by mistahj67** [Bungie-net/api#775]
Enhancement for the Clan endpoints to provide webhooks subscription option for events that occur:
Member Join
Member Leave
Member Invited
Etc.
Could subscribe Bungie application and provide url to send to. Perhaps similar to the auth callback url.

**Comment by vthornheart-bng** [comment]
Yeah, I dig that idea!  It'll be on the distant wishlist as I don't know when we'll have time to do it, but this is one of the very few and very rare instances where we actually do have a way to get this info proactively pushed to us by the game, and as a result we actually could send pushes outward.  (almost all destiny data is pull-only, making webhooks infeasible... but some clan actions are an exception)

**Comment by nine13tech** [comment]
Could that also include the "Request to Join"
